caddyhttp/httpserver/plugin.go

```diff
@@ -410,6 +410,9 @@ var directives = []string{
 	"gzip",
 	"header",
 	"errors",
+	"filter", // github.com/echocat/caddy-filter - Required at this position because if it is
+	//                                             enabled it captures the response of following
+	//                                             middlewares and modifies it.
```

Comment:
No problem. I just think it could be a good idea to left a hint for this strange thing - that not someones reorders this and we trap in new trouble. :)